_data/banking.yml

```
@@ -1444,32 +1808,42 @@ websites:
       - phone
       - hardware
     doc: https://www.wellsfargo.com/privacy-security/advanced-access
+    regions:
+      - us
 
   - name: Wescom Credit Union
     url: https://www.wescom.org/
     twitter: Liz_Wescom
     facebook: WescomCreditUnion
     img: wescom.png
+    regions:
+      - us
 
   - name: Westminster Savings Credit Union (WSCU)
     url: https://www.wscu.com/
     twitter: wscu
     facebook: westminstersavings
     img: wscu.png
+    regions:
+      - us
```

```suggestion
      - ca
```
WSCU is a Canadian credit union. They've also [merged](https://futurestrong.ca/) with [Prospera Credit Union](https://prospera.ca/). They're insured under CUDIC.